docs/guides/data-writes.md

```diff
@@ -516,9 +516,17 @@ From your components perspective, all that happened was the `useTransition` hook
 
 ## See also
 
-- [Form](../api/remix#form)
-- [useTransition](../api/remix#usetransition)
-- [Actions](../api/conventions#action)
-- [Loaders](../api/conventions#loader)
-- [`useSubmit()`](../api/remix#usesubmit)
-- [`useFetcher()`](../api/remix#usefetcher)
+- [Form][form-2]
+- [useTransition][use-transition]
+- [Actions][actions]
+- [Loaders][loaders]
+- [`useSubmit()`][use-submit]
+- [`useFetcher()`][use-fetcher]
+
+[form]: ../api/remix#form
+[use-submit]: ../api/remix#usesubmit
+[use-fetcher]: ../api/remix#usefetcher
+[form-2]: ../api/remix#form
```

Comment:
This is the same as the first link.

Probably a bit hard, but could we maybe keep it at 1 reference?

Reply:
definitely should be one, checking 🙂